### ISSUE TITLE
When app is killed

### ISSUE THREAD

**Issue by falay26** [react-native-geolocation/react-native-geolocation#154]
Can this library allow me to get user's location data (even when app is killed or backgrounded) and store it to my database?

**Comment by DanielGannage** [comment]
Did you get an answer to this? I would like to use geolocation in the background as well. Not sure if this is the right module for that...

**Comment by dev-andremonteiro** [comment]
@falay26 @DanielGannage Did you find any library that is capable of doing this ?

**Comment by DanielGannage** [comment]
> @falay26 @DanielGannage Did you find any library that is capable of doing this ?

Yes, react native background actions will allow me to run it in the background.

**Comment by jamalSlider** [comment]
+1
